static/bidder-params/interactiveoffers.json

```diff
@@ -4,10 +4,10 @@
   "description": "A schema which validates params accepted by Interactive Offers adapter",
   "type": "object",
   "properties": {
-    "pubid": {
-      "type": "integer",
-      "description": "The publisher id"
+    "partnerId": {
+      "type": "string",
+      "description": "The partners id"
     }
   },
-  "required": ["pubid"]
```

Comment:
This is a difficult thing to pull off. We made a design choice that any request that has any bidder that fails the JSON schema will invalidate the entire request. The idea was that this would only happen when a publisher creates/updates their prebid config. When they do this, they should be looking to see if prebid is returning any bids, but probably aren't sensitive to if all bidders are bidding. They would otherwise not notice that a bidder is misconfigured until at some point they look at their analytics and notice the bidder is not bidding on a particular page or configuration setup. We think the current design choice is the better option to ensure everyone has a chance to bid.

The recommended path here is to drop `pubid` as a required parameter, but leave `partnerId` as optional. The adapter can then return an error explaining the issue with the request. If a request passes the JSON schema, but a bidder returns an error explaining the issue when `MakeRequests` is called. This will allow the other configured bidders to bid rather than invalidating the entire request. At a later point in time, when the publishers have all had a chance to update to the new schema, we can make it required here. We just don't want all the publisher who are currently using your adapter to suddenly have all their prebid configs invalidated by surprise.